manifest_staging/charts/aad-pod-identity/values.yaml

```diff
@@ -211,19 +211,30 @@ rbac:
   allowAccessToSecrets: true
 
 # Create azure identities and bindings
-azureIdentities: []
-  # - name: "azure-identity"
+# This is a map with the AzureIdentityName being the key and the rest of the blob as value in accordance
+# to helm best practices: https://helm.sh/docs/chart_best_practices/values/#consider-how-users-will-use-your-values
+azureIdentities:
+  # "azure-identity":
   #   # if not defined, then the azure identity will be deployed in the same namespace as the chart
   #   namespace: ""
-  #   # type 0: MSI, type 1: Service Principal
+  #   # type 0: MSI, type 1: Service Principal, type 2: Service principal with certifcate
   #   type: 0
   #   # /subscriptions/subscription-id/resourcegroups/resource-group/providers/Microsoft.ManagedIdentity/userAssignedIdentities/identity-name
+  #   # Required for type 0
   #   resourceID: ""
+  #   # Required for type 0, 1 and 2
   #   clientID: ""
+  #   # Required for type 1 and 2
+  #   tenantID: ""
+  #   # Required for type 1 and 2
+  #   clientPassword: {}
```

Comment:
```suggestion
  #   clientPassword:
  #     name: ""
  #     namespace: "" 
```

Reply:
`clientPassword` is not a map. It is [types.NamespacedName](https://pkg.go.dev/k8s.io/apimachinery/pkg/types#NamespacedName). I've updated the PR with the format.